ogusa/constants.py

```diff
@@ -1,10 +1,13 @@
 import taxcalc
+import os
 
 SHOW_RUNTIME = False  # Flag to display RuntimeWarnings when run model
 
 REFORM_DIR = "OUTPUT_REFORM"
 BASELINE_DIR = "OUTPUT_BASELINE"
 
+CODE_PATH = os.path.abspath(os.path.dirname(__file__))
```

Comment:
@jdebacker. I'll test this line of code in `constants.py`, but I think it will break any runs in a Jupyter notebook.

Reply:
Seems to work fine for me.  And [Tax-Calculator does the same](https://github.com/PSLmodels/Tax-Calculator/blob/dd983415b96393ee9434c9d457b4ec9c291df58a/taxcalc/records.py#L121) and I've had not issues with it in notebooks locally or on Google Colab.